test/integration/__snapshots__/integration.test.ts.snap

```diff
@@ -425,11 +444,146 @@ export interface TableTypes {
 export const tables = {
   comment,
   doc,
+  table_with_underscores,
   users,
 }
 "
 `;
 
+exports[`Integration tests Schema generation pg-to-sql with camelCase 1`] = `
+"/* tslint:disable */
+/* eslint-disable */
+
+import { CommentMetadata } from \\"./pg-to-ts-json-types\\";
+
+
+export type Json = unknown;
+export type CommentStatusEnum = 'archived' | 'complete' | 'deleted' | 'pending';
+
+// Table public.comment
+/** Variant-level comments */
+export interface PublicComment {
+  id: string;
+  docId: string;
+  authorId: string;
+  createdAt: string | null;
+  modifiedAt: string | null;
+  /** Additional comment info @type {CommentMetadata} */
+  metadata: CommentMetadata | null;
+  /** Content of the comment, formatted with Markdown. May contain @mentions. */
+  contentMd: string;
+  /** List of statuses; Just an array for testing! */
+  statuses: CommentStatusEnum[] | null;
+}
+/** Variant-level comments */
+export interface PublicCommentInput {
+  id?: string;
+  docId: string;
+  authorId: string;
+  createdAt?: string | null;
+  modifiedAt?: string | null;
+  /** Additional comment info @type {CommentMetadata} */
+  metadata?: CommentMetadata | null;
+  /** Content of the comment, formatted with Markdown. May contain @mentions. */
+  contentMd: string;
+  /** List of statuses; Just an array for testing! */
+  statuses?: CommentStatusEnum[] | null;
+}
+const public_comment = {
+  tableName: 'public.comment',
+  columns: ['id', 'docId', 'authorId', 'createdAt', 'modifiedAt', 'metadata', 'contentMd', 'statuses'],
+  requiredForInsert: ['docId', 'authorId', 'contentMd'],
+  primaryKey: 'id',
+  foreignKeys: {
+    doc_id: { table: 'doc', column: 'id' },
+    author_id: { table: 'users', column: 'id' },
+  },
+} as const;
+
+// Table public.doc
+export interface PublicDoc {
+  id: string;
+  createdBy: string;
+  title: string | null;
+  contents: string | null;
+}
+export interface PublicDocInput {
+  id?: string;
+  createdBy: string;
+  title?: string | null;
+  contents?: string | null;
+}
+const public_doc = {
+  tableName: 'public.doc',
+  columns: ['id', 'createdBy', 'title', 'contents'],
+  requiredForInsert: ['createdBy'],
+  primaryKey: 'id',
+  foreignKeys: { created_by: { table: 'users', column: 'id' }, },
+} as const;
+
+// Table public.table_with_underscores
+export interface PublicTableWithUnderscores {
+  columnWithUnderscores: string;
+}
+export interface PublicTableWithUnderscoresInput {
+  columnWithUnderscores: string;
+}
+const public_table_with_underscores = {
```

Comment:
I believe this was the case you mentioned in https://github.com/danvk/pg-to-ts/pull/14/files#r841181807? It seems to be OK here!